kubectl-cloudflow/config/config.go

```diff
@@ -694,13 +694,40 @@ func addCommandLineArguments(spec cfapp.CloudflowApplicationSpec, config *Config
 	return config
 }
 
+// CreateAppInputSecret creates input secret
 func CreateAppInputSecret(spec *cfapp.CloudflowApplicationSpec, config *Config) (*corev1.Secret, error) {
 	secretMap := make(map[string]string)
 	secretMap["secret.conf"] = config.String()
 	secret := CreateInputSecret(spec.AppID, secretMap)
 	return secret, nil
 }
 
+// CreateEmptyStreamletSecret creates empty streamlet secret
+func CreateEmptyStreamletSecret(appID string, name string, ownerReference metav1.OwnerReference) *corev1.Secret {
+	labels := cfapp.CreateLabels(appID)
+	labels["com.lightbend.cloudflow/app-id"] = appID
+	labels["com.lightbend.cloudflow/created-at"] = fmt.Sprintf("%d", time.Now().UnixNano())
+	labels["com.lightbend.cloudflow/streamlet-name"] = name
+	labels["com.lightbend.cloudflow/config-format"] = "config"
+	secret := &corev1.Secret{
+		Type: corev1.SecretTypeOpaque,
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: appID,
+			Labels:    labels,
+		},
+	}
+	secretMap := make(map[string]string)
+	secretMap["secret.conf"] = "{}"
+	secretMap["runtime-config.conf"] = "{}"
```

Comment:
If I read this change correctly, this is the core part of it, have you tested if just adding those defaults to `createOrUpdateAppInputSecret` make it?

Reply:
n streamlets -> n empty secrets. 1 app secret.